### PR TITLE
Javascript javascript2 week3

### PR DESCRIPTION
**Pull request by melo4yxa77796**
my memory game for week 3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,16 +9,22 @@
   <body>
     <h1>Memory game</h1>
     <div class="wrapper">
+        <p id="results">
```
Comment (vdaguenet) [inline index.html:12]: Be careful with indentation here.

File: index.js [hunk 2]
```diff
@@ -34,54 +23,108 @@ const emojis = [
...
-});
+  const stopButton = document.getElementById("stop");
+
+  startButton.addEventListener("click", startGame);
+  stopButton.addEventListener("click", stopGame);
+  stopButton.disabled = true;
+
+  let moveCounter = 0;
+
+  function initializeGame() {
```
Comment (vdaguenet) [inline index.js]: This function does a lot. To improve the readability of your code, you could split it into smaller functions. For example, `createGameBoard()` would shuffle emojis and display the grid; `onCardClick()` would flip the card and increment the counter; etc.

**Review by vdaguenet — APPROVED** [review]
It works well 👍🏻 

A suggestion that you could implement, or not, it's up to you, would be to disable clicking on cards while the game is not started.

File: index.html (hunk 1 above)
Comment (melo4yxa77796) [reply index.html:12]: Could you please explain what the problem is if the "id" is called that, I don't understand

File: index.js (hunk 2 above)
Comment (melo4yxa77796) [reply index.js]: i made

File: index.html (hunk 1 above)
Comment (vdaguenet) [reply index.html:12]: I did not talk about ID but indentation, meaning the amount of spaces at the beginning of the line. They are irregular in this section of your file: the `<p>` line 12 should be indented with 2 spaces less, the `<div>` line 15 should be aligned with the `div` on the previous line, etc.

**Pull request merged**